### DISCUSSION TITLE
Transfer my BTC from Copay to Bitpay>

### DISCUSSION THREAD

**Discussion by Jusdaboyz01** [community/community#54422] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

How do I merge my BTC in my Copay wallet over to my Bitpay wallet

**Comment by samus-aran** [comment, marked as answer]
Hi @Jusdaboyz01 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
